templates/build_matrix

```diff
@@ -18,3 +18,5 @@ linux arm
 linux arm64
 windows 386
 windows amd64
+windows amd64-v2
+windows amd64-v3
```

Comment:
@Jorropo mind moving this unrelated change to separate PR? 

I feel wee need to discuss if there is any point in additional build time for windows, if IPFS DEsktop always uses the `amd64` one.

Reply:
I was expecting to update ipfs-update and maybe ipfs-desktop later.
Anyway will done: https://github.com/ipfs/distributions/pull/791